### ISSUE TITLE
"Create a new changeset?" prompt occurs when previous changeset is near

### ISSUE THREAD

**Issue by SomeoneElseOSM** [systemed/potlatch3#29]
After saving for the first time https://www.openstreetmap.org/changeset/116758849 , which I save again I get a message "Your previous changeset is still open and is a long way away. Create a new changeset?".  At the time that that message appears, the first change in that changeset is just down the road?

**Comment by systemed** [comment]
Hm, haven't seen that - if you get steps to reproduce then let me know.

**Comment by SomeoneElseOSM** [comment]
It seems to happen "every time" (in that I've run it twice and it's happened twice).  It happens on Windows 10 and Linux under "win64" (which pretends to be Windows 7).  I downloaded and unziped https://www.systemed.net/potlatch/download/Potlatch.zip and ran Potlatch.exe from in there.  The previous version still seems to be installed and run from ...Downloads\Potlatch\Potlatch .  Latest changeset https://www.openstreetmap.org/changeset/116760083 .

**Comment by systemed** [comment]
Odd. I'll see if I can reproduce it.

**Comment by BikePC** [comment]
I can confirm & reproduce the issue @SomeoneElseOSM describes above. Windows10 64bit. Potlatch.exe from downloaded & unzipped Potlatch..zip.

**Comment by DaveF63** [comment]
After the updates shown in #47 this problem has reoccurred for me. 
I previously solved it by installing an older version of P3 
Is there one which will still allow the current Auth settings?

**Comment by systemed** [comment]
No, there isn't. I'll take a look at it when I get a spare minute (but that won't be for a few weeks I'm afraid).
